jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/AdministrationControllerTest.java

```diff
@@ -318,22 +319,48 @@ public void groupAdministrationPageShouldContainListOfGroups() throws Exception
                 .andExpect(model().attribute("groups", expected));
     }
 
+    private List<UserDto> buildTestUserDtoList() {
+        List<UserDto> result = new ArrayList<>();
+        result.add(new UserDto("user1", "user1@mail.ru", 1L));
+        result.add(new UserDto("user2", "user2@mail.ru", 2L));
+        result.add(new UserDto("user3", "user3@mail.ru", 3L));
+        return result;
+    }
+
+    private Group buildTestGroup() {
+        Group result = new Group("Group name");
+        result.setId(1L);
+        return result;
+    }
+
+    private Map<String, Object> performTestRequest() throws Exception {
+         return this.mockMvc
+                 .perform(get("/group/1").accept(MediaType.TEXT_HTML))
+                 .andReturn()
+                 .getModelAndView()
+                 .getModel();
+    }
+
+    private void prepareGroupServiceMock(Group expectedGroup, List<UserDto> expectedGroupUsers) throws org.jtalks.common.service.exceptions.NotFoundException {
```

Comment:
It doesn't make sense mixing mocking and `MockMvc`. And we prefer not to use mocking at all..